.github/workflows/build-and-publish.yaml

```diff
@@ -12,8 +12,40 @@ env:
   GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
 jobs:
+  validate:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          path: proto
+
+      - name: Install ZCBOR
+        run: sudo apt update && sudo apt install python3-pip -y && pip3 install zcbor
+
+      - name: Validate CDDL and convert JSON
+        working-directory: proto/hello/firmware
+        run: |
```

Comment:
That should be a script file so it can be executed without using GitHub Actions.

Reply:
it now lives in hello/firmwarevalidate.sh